### PR TITLE
Completed user story user roles

### PR DESCRIPTION
**Pull request by blunce24**
I was able to create a new user in the console, and it had the standard role by default. I was also able to edit another user's wiki.

### CODE REVIEW

File: app/policies/application_policy.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
+class ApplicationPolicy
```
Comment (bmneely) [inline app/policies/application_policy.rb]: Nice job, what do you think of Pundit so far?

File: app/views/wikis/index.html.erb [hunk 2]
```diff
@@ -9,7 +9,7 @@
             <%= link_to wiki.title, wiki %>
           </h4>
           <small>
-             submitted <%= time_ago_in_words(post.created_at) %> ago by <%= post.user.name %>
+             submitted <%= time_ago_in_words(wiki.created_at) %> ago by <%= wiki.user.email %>
```
Comment (bmneely) [inline app/views/wikis/index.html.erb]: 👍

File: spec/models/user_spec.rb [hunk 3]
```diff
@@ -1,5 +1,89 @@
...
+
+      it "returns false for #premium?" do
+        expect(@user.premium?).to be_falsey
+      end
+
+      it "returns true for #admin?" do
+        expect(@user.admin?).to be_truthy
+      end
+    end
+  end
```
Comment (bmneely) [inline spec/models/user_spec.rb]: Beautiful tests!

File: app/policies/application_policy.rb (hunk 1 above)
Comment (blunce24) [inline app/policies/application_policy.rb]: I like it - it seems like a very straightforward way to define authorization roles. It'll be interesting to work with it further.

**Pull request merged**